src/components/character-count/index.md.njk

```diff
@@ -97,6 +97,16 @@ For example, ‘Enter a summary’.
 Say ‘[whatever it is] must be [number] characters or fewer’.<br>
 For example, ‘Summary must be 400 characters or fewer’.
 
+### Consider the effect of emojis and accented characters
+
+Using emojis and accented characters can affect character counts in a way you might not want.
+
+The character count component uses the `string.length` property to work out how long the user input is.
```

Comment:
Should we link to an explanation of `string.length`? A [related issue](https://github.com/alphagov/govuk-frontend/issues/1104#issue-393476982) has some links - but I don't know whether we want to include any of them within Design System guidance.